tensorboard/defs/internal/html.bzl

```diff
@@ -15,7 +15,10 @@
 """Rule for building the HTML binary."""
 
 load("@io_bazel_rules_closure//closure:defs.bzl", "closure_js_aspect")
-load("@io_bazel_rules_closure//closure/private:defs.bzl", "WebFilesInfo", "collect_runfiles", "extract_providers", "long_path", "unfurl")  # buildifier: disable=bzl-visibility
+load("@io_bazel_rules_closure//closure/private:defs.bzl", "WebFilesInfo", "collect_runfiles", "long_path", "unfurl")  # buildifier: disable=bzl-visibility
+
+def extract_providers(deps, provider):
```

Comment:
Maybe let's add a comment for removing these once the internal rules_closure repo gets synced with the OSS, so we can import this function again?

Reply:
Good suggestion, done.